src/functions-reference/distributions_over_unbounded_vectors.Rmd

```diff
@@ -593,6 +594,139 @@ freedom nu, locations mu, and scale matrix Sigma; may only be used in
 transformed data andgenerated quantities blocks
 `r since("2.18")`
 
+## Multivariate Student-t distribution, Cholesky parameterization {#multi-student-t-cholesky-fun}
+
+### Probability density function
+
+If $K \in \mathbb{N}$, $\nu \in \mathbb{R}^+$, $\mu \in \mathbb{R}^K$,
+and $\Sigma \in \mathbb{R}^{K \times K}$ is symmetric and positive
+definite, then for $y \in \mathbb{R}^K$, \[ \begin{array}{l}
```

Comment:
[optional]
I'd prefer to see the display math set out with `\[` and `\]` on their own lines with the inside set out for readability.